announcements/_posts/2019-11-01-w10-linux-gaming.md

```diff
@@ -0,0 +1,33 @@
+---
+author: Henry Keena (@htk4363)
+title: "Gaming in Linux"
+layout: post-event
+date-start: "2019-11-1 16:30"
+date-end: "2019-11-1 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey Ritluggers,
+
+This friday, our feature presentation we will be about gaming in linux.
+We will be going in depth into the linux gaming scene, what games you can play on linux, as well as tools and applications you can use to play games on your distro of choice!
+If you're interested in gaming, come learn about how you can make your linux box your new gaming rig.
+
+We will be ordering t-shirts within the next few weeks.
+If you are interested in buying a RITlug t-shirt, fill out [this form](https://forms.gle/8zDpxyxFy9URTw71A).
+The T-shirts will cost $20. 
+
+GCCIS is hosting a Halloween costume tomorrow(10/31/2019).
```

Comment:
"Halloween costume *contest*", but at this point, it's not tomorrow anymore :sweat_smile: